dWorldServer/WorldServer.cpp

```diff
@@ -1284,12 +1285,14 @@ void WorldShutdownProcess(uint32_t zoneId) {
 }
 
 void WorldShutdownSequence() {
-	if (Game::shouldShutdown || worldShutdownSequenceComplete) {
+	Game::shouldShutdown = true;
```

Comment:
This is atomic because in multithreaded handling, the main loop will start the shutdown, and setting this to true across a thread will have no effect.